tests/main/views/test_direct_award.py

```diff
@@ -467,6 +468,98 @@ def test_end_search_redirects_to_project_page(self):
         assert res.location.endswith('/buyers/direct-award/g-cloud/projects/1')
 
 
+class TestDirectAwardAwardContract(TestDirectAwardBase):
+    def setup_method(self, method):
+        super().setup_method(method)
+        self.data_api_client.get_direct_award_project.return_value['project']['lockedAt'] = \
+            "2017-08-30T07:49:26.677778Z"
+
+    def test_award_contract_page_renders(self):
+        self.login_as_buyer()
+
+        res = self.client.get('/buyers/direct-award/g-cloud/projects/1/did-you-award-contract')
+        assert res.status_code == 200
+
+        doc = html.fromstring(res.get_data(as_text=True))
+        assert len(doc.xpath(
+            '//h1[contains(normalize-space(), "Did you award a contract for \'My procurement project\'?")]')) == 1
```

Comment:
You can also use variables in the xpath strings if you want to avoid escaping strings. I think you can do something like the below. Just FYI, no need to change this.

```
doc.xpath('//h1[contains(normalize-space(), "$text")]',
          text="Did you award a contract for 'My procurement project'?")
```